server/controllers/users/index.js

```diff
@@ -0,0 +1,28 @@
+import { Users } from "../../database/models";
+import db from "../../database";
+import { badRequest, okResponse } from "../../helpers/response";
+import { Admin, User } from "../../helpers/roles";
+import bcrypt from "../../helpers/bcrypt";
+import { createToken } from "../../helpers/jwt";
+
+export default class UserController {
+  static async auth(req, res) {
```

Comment:
Reduce the use of `static` function, try to instantiate the class during export the instance of the class.

Reply:
In order to remove the static functions here in nodejs, I had to export default the controllers' functions I hope it's okay